moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -432,21 +432,29 @@ void MotionPlanningFrame::collisionObjectChanged(QListWidgetItem* item)
 /* Receives feedback from the interactive marker and updates the shape pose in the world accordingly */
 void MotionPlanningFrame::imProcessFeedback(visualization_msgs::msg::InteractiveMarkerFeedback& feedback)
 {
+  if (!planning_display_->getPlanningSceneRO()->knowsFrameTransform(feedback.header.frame_id))
+  {
+    RCLCPP_ERROR_STREAM(LOGGER,
+                        "Frame `" << feedback.header.frame_id << "` unknown doesn't exists in the planning scene");
+  }
+  Eigen::Isometry3d fixed_frame_t_scene_marker;
```

Comment:
hmm... what is the situation in ROS 1? Wouldn't this be needed there as well?

Reply:
I tested it in ROS1, this bug doesn't exist there :confused:

Reply:
How is the frame being handled in ROS 1 then? To me this looks like a bug that was introduced in a different place during conversion.